delta-app/src/main/java/io/cdap/delta/app/service/AssessmentHandler.java

```diff
@@ -191,6 +195,6 @@ private void respond(String namespaceName, HttpServiceResponder responder, Names
     /**
      * Create the response that should be returned by the endpoint.
      */
-    void respond(Namespace namespace) throws Exception;
+    void respond(DraftService draftService, Namespace namespace) throws Exception;
```

Comment:
Not quite sure what the question is, but what we're trying to avoid is having any application logic in the handler. The handler should only be responsible for decoding request bodies and things of that nature. This way the DraftService can be used in the handler, in a unit test, in some other service, or in some other handler, and we're ensured that it's the same logic.